test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/BasicAuthTests.cs

```diff
@@ -26,6 +29,8 @@ namespace Microsoft.Health.Fhir.Tests.E2E.Rest
     [HttpIntegrationFixtureArgumentSets(DataStore.CosmosDb, Format.Json)]
     public class BasicAuthTests : IClassFixture<HttpIntegrationTestFixture>
     {
+        private static readonly Regex WwwAuthenticatePattern = new Regex(@".*authorization_uri\s*=\s*(?<authorization_uri>[^\s,]*)?, resource_id=\""*(?<resource_id>[^""]+)\""*, realm=\""*(?<realm>[^""]+)\""*", RegexOptions.IgnoreCase);
```

Comment:
Not sure it is hugely important, but the header you are generating does not have '"' around the `resource_id` or the `realm` string. But this regex optionally matches them. It should maybe also be `\""?` (zero or one).

Reply:
This regex was a bit more robust than needed for the test. I trimmed it down to be more in line with what we actually expect the header to be.